docs/BR.md

```diff
@@ -994,25 +1026,88 @@ Databases maintained by the CA, its owner, or its affiliated companies do not qu
+
+Results or information obtained from one Network Perspective MUST NOT be reused or cached when performing validation through subsequent Network Perspectives (e.g., different Network Perspectives cannot rely on a shared DNS cache to prevent an adversary with control of traffic from one Network Perspective from poisoning the DNS cache used by other Network Perspectives). The network infrastructure providing Internet connectivity to a Network Perspective MAY be administered by the same organization providing the computational services required to operate the Network Perspective. All communications between a remote Network Perspective and the CA MUST take place over an authenticated and encrypted channel relying on modern protocols (e.g., over HTTPS).
+
+A Network Perspective MAY use a recursive DNS resolver that is NOT co-located with the Network Perspective. However, the DNS resolver used by the Network Perspective MUST fall within the same Regional Internet Registry service region as the Network Perspective relying upon it. Furthermore, for any pair of DNS resolvers used on a Multi-Perspective Issuance Corroboration attempt, the straight-line distance between the two States, Provinces, or Countries the DNS resolvers reside in MUST be at least 500 km. The location of a DNS resolver is determined by the point where unencapsulated outbound DNS queries are first handed off to the network infrastructure providing Internet connectivity to that DNS resolver.
+
+CAs MAY immediately retry Multi-Perspective Issuance Corroboration using the same validation method or an alternative method (e.g., a CA can immediately retry validation using "Email to DNS TXT Contact" if "Agreed-Upon Change to Website - ACME" does not corroborate the outcome of Multi-Perspective Issuance Corroboration). When retrying Multi-Perspective Issuance Corroboration, CAs MUST NOT rely on corroborations from previous attempts. There is no stipulation regarding the maximum number of validation attempts that may be performed in any period of time.
+
+The "Quorum Requirements" Table describes quorum requirements related to Multi-Perspective Issuance Corroboration. Network Perspectives are considered distinct when the straight-line distance between the two States, Provinces, or Countries they reside in is at least 500 km. Network Perspectives are considered "remote" when they are distinct from the Primary Network Perspective and the other Network Perspectives represented in a quorum.
+
+A CA MAY reuse corroborating evidence for CAA record quorum compliance for a maximum of 398 days. After issuing a Certificate to a domain, remote Network Perspectives MAY omit retrieving and processing CAA records for its subdomains in subsequent Certificate requests from the same Applicant for up to a maximum of 398 days.
```

Comment:
Hi Doug!

I often find example scenarios helpful in cases such as this. Please find one below:

Note: for this example, assume DCV data reuse is set to 90 days by the CA.

**Today:**
- An applicant requests a certificate for the dNSname "ilovetls.com" using “Agreed-Upon Change to Website - ACME” (TLS BR 3.2.2.4.19).
- Primary DCV initiated: Their ACME client demonstrates control over the requested domain in accordance with TLS 3.2.2.4.19, and this is verified by the CA from its primary data center.
- MPIC DCV initiated: The demonstration of control over the requested domain would be corroborated by [N] remote network perspectives in accordance with the proposed TLS BR 3.2.2.9.
- MPIC CAA initiated: Successful MPIC CAA checks must take place for the requested domain in accordance with the proposed TLS BR 3.2.2.9.
- Primary CAA check initiated: The CA checks CAA for the the requested domains in accordance with TLS BR 3.2.2.8.
- Assuming no issues with the above, the certificate is issued.

**100 days from now:**
- The same applicant requests another certificate for the dNSname "ilovetls.com" using “Agreed-Upon Change to Website - ACME” (TLS BR 3.2.2.4.19).
- Primary DCV initiated: Their ACME client demonstrates control over the requested domain in accordance with TLS 3.2.2.4.19, and this is verified by the CA from its primary data center.
- MPIC DCV initiated: The demonstration of control over the requested domain would be corroborated by [N] remote network perspectives in accordance with the proposed TLS BR 3.2.2.9.
- MPIC CAA optional: Given it's been less than 398-days since the last successful MPIC CAA check and the same applicant is requesting the certificate, this may be skipped as described in the proposed 3.2.2.9.
- Primary CAA check initiated: The CA checks CAA for the the requested domains in accordance with TLS BR 3.2.2.8.
- Assuming no issues with the above, the certificate is issued.

Note: there's flexibility to the sequencing above (e.g., the MPIC lookups for DCV and CAA may take place at the same time), this is just one way of doing it.  

**To directly address your question:** Yes [but only for the remainder of the permitted 398-day reuse period for MPIC CAA data - and if the Applicant is determined to be the same as the original request.]

Does this address your question?

Reply:
Thanks a lot Ryan for this very illustrative example! I understand where it comes from... it just seems a bit counter-intuitive. ;-) I probably missed the discussion that led to this wording. Any pointers where I can read up?
Thx!